[2023-06-29T22:01:03.951+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Oracle_to_Filestore.execute_function_task manual__2023-06-29T22:01:02.780733+00:00 [queued]>
[2023-06-29T22:01:03.955+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Oracle_to_Filestore.execute_function_task manual__2023-06-29T22:01:02.780733+00:00 [queued]>
[2023-06-29T22:01:03.955+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-29T22:01:03.962+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): execute_function_task> on 2023-06-29 22:01:02.780733+00:00
[2023-06-29T22:01:03.966+0000] {standard_task_runner.py:57} INFO - Started process 3773 to run task
[2023-06-29T22:01:03.969+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Oracle_to_Filestore', 'execute_function_task', 'manual__2023-06-29T22:01:02.780733+00:00', '--job-id', '591', '--raw', '--subdir', 'DAGS_FOLDER/Python-Delta-Lake/Dags/SparkOracle2FileStoreDag.py', '--cfg-path', '/tmp/tmpd63jqwzd']
[2023-06-29T22:01:03.970+0000] {standard_task_runner.py:85} INFO - Job 591: Subtask execute_function_task
[2023-06-29T22:01:04.010+0000] {task_command.py:410} INFO - Running <TaskInstance: Oracle_to_Filestore.execute_function_task manual__2023-06-29T22:01:02.780733+00:00 [running]> on host a35364001907
[2023-06-29T22:01:04.060+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='phomsophaN@saccounty.net' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Oracle_to_Filestore' AIRFLOW_CTX_TASK_ID='execute_function_task' AIRFLOW_CTX_EXECUTION_DATE='2023-06-29T22:01:02.780733+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-29T22:01:02.780733+00:00'
[2023-06-29T22:01:04.060+0000] {logging_mixin.py:149} INFO - SparkSession Not found, creating one
[2023-06-29T22:01:04.061+0000] {logging_mixin.py:149} INFO - setting up configured spark connection Cluster
[2023-06-29T22:01:04.061+0000] {logging_mixin.py:149} INFO - Initializing support for delta lake spark context
[2023-06-29T22:01:07.062+0000] {logging_mixin.py:149} INFO - Returning Spark Context
[2023-06-29T22:01:07.063+0000] {client.py:192} INFO - Instantiated <InsecureClient(url='http://namenode:50070')>.
[2023-06-29T22:01:07.064+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:01:07.077+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:01:07.077+0000] {logging_mixin.py:149} INFO - HDFS Detected and Connected
[2023-06-29T22:01:07.077+0000] {logging_mixin.py:149} INFO - Spark Connection check - session found
[2023-06-29T22:01:07.687+0000] {logging_mixin.py:149} INFO - Attempt to read Msql Table by query: select * from DeltaLake.OracleTablesLog
        where OracleRows < 1000000 AND OracleRows > 10000 AND OracleSizeInMB < 4000 AND CountyFilter IS NOT NULL and PrimaryKeyColumn !=''
[2023-06-29T22:01:11.282+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.10/site-packages/pyspark/sql/pandas/conversion.py:248: FutureWarning: Passing unit-less datetime64 dtype to .astype is deprecated and will raise in a future version. Pass 'datetime64[ns]' instead
  series = series.astype(t, copy=False)

[2023-06-29T22:01:11.282+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.10/site-packages/pyspark/sql/pandas/conversion.py:248: FutureWarning: Passing unit-less datetime64 dtype to .astype is deprecated and will raise in a future version. Pass 'datetime64[ns]' instead
  series = series.astype(t, copy=False)

[2023-06-29T22:01:11.287+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.10/site-packages/pyspark/sql/pandas/conversion.py:248: FutureWarning: Passing unit-less datetime64 dtype to .astype is deprecated and will raise in a future version. Pass 'datetime64[ns]' instead
  series = series.astype(t, copy=False)

[2023-06-29T22:01:11.287+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.10/site-packages/pyspark/sql/pandas/conversion.py:248: FutureWarning: Passing unit-less datetime64 dtype to .astype is deprecated and will raise in a future version. Pass 'datetime64[ns]' instead
  series = series.astype(t, copy=False)

[2023-06-29T22:01:11.288+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.10/site-packages/pyspark/sql/pandas/conversion.py:248: FutureWarning: Passing unit-less datetime64 dtype to .astype is deprecated and will raise in a future version. Pass 'datetime64[ns]' instead
  series = series.astype(t, copy=False)

[2023-06-29T22:01:11.289+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.10/site-packages/pyspark/sql/pandas/conversion.py:248: FutureWarning: Passing unit-less datetime64 dtype to .astype is deprecated and will raise in a future version. Pass 'datetime64[ns]' instead
  series = series.astype(t, copy=False)

[2023-06-29T22:01:11.290+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.10/site-packages/pyspark/sql/pandas/conversion.py:248: FutureWarning: Passing unit-less datetime64 dtype to .astype is deprecated and will raise in a future version. Pass 'datetime64[ns]' instead
  series = series.astype(t, copy=False)

[2023-06-29T22:01:11.299+0000] {logging_mixin.py:149} INFO - ['PR_LRS.FOSTER_FAM_AGNC', 'PR_LRS.EBT_EXPG_EXCPT', 'PR_LRS.ENCL_ESTIMATE', 'PR_LRS.UNIT', 'PR_LRS.IEVS_SAVE_EXCEPT', 'PR_LRS.WTW_24_MONTH_SIGN_DATE', 'PR_LRS.RULES_ADMIN', 'PR_LRS.COUNTY_PARAMTR_ADMIN_HST', 'PR_LRS.C4Y_PREG', 'PR_LRS.IEVS_PRISON_MATCH', 'PR_LRS.ORG_ACCT', 'PR_LRS.TRAIN_PGM', 'PR_LRS.IFD_SSI', 'PR_LRS.C4Y_PERS_NON_COMPL', 'PR_LRS.POS', 'PR_LRS.ADH_SUMM', 'PR_LRS.WDTIP_PGM_PARTICPTN', 'PR_LRS.HEAR_SUMM', 'PR_LRS.WDTIP_ALERT', 'PR_LRS.IHSS_REFRL', 'PR_LRS.SFIS_COUNTY_INFO', 'PR_LRS.STAFF', 'PR_LRS.MEDS_ALERT_CONFIG', 'PR_LRS.BATCH_JOB', 'PR_LRS.IHSS_CASE', 'PR_LRS.RULES_ADMIN_HST', 'PR_LRS.TAX_INTRCPT', 'PR_LRS.WTW_24_MONTH_SIGN_DATE_HST', 'PR_LRS.COUNTY_PARAMTR_ADMIN', 'PR_LRS.OCAT', 'PR_LRS.WDTIP_EXCEED_CLOCK', 'PR_LRS.AUTO_ACTN', 'PR_LRS.IFD_DUPL_DETL', 'PR_LRS.EBT_FRAUD_ACTIV', 'PR_LRS.GA_GR_TIME_LIMIT', 'PR_LRS.GRP_HOMES', 'PR_LRS.ASSET_VERIF', 'PR_LRS.FUND_CODE_MAP', 'PR_LRS.C4Y_OTHER_PGM_ASSIST', 'PR_LRS.IEVS_FF_MEDS', 'PR_LRS.COLLAB_CONTRACT', 'PR_LRS.WDTIP_APPRCH_CLOCK', 'PR_LRS.EXTRNL_STAFF_COUNTY_STAT', 'PR_LRS.VEND_IDENTIF', 'PR_LRS.OCAT_PERS', 'PR_LRS.MEDS_ALERT_CONFIG_HST', 'PR_LRS.WPR_SAMPLE_CASE', 'PR_LRS.PERS_TIME_TRACK_DETL']
[2023-06-29T22:01:11.300+0000] {logging_mixin.py:149} INFO - processing 1 out of 48
[2023-06-29T22:01:11.300+0000] {logging_mixin.py:149} INFO - Attempting to get lowest value by column ID on table PR_LRS.FOSTER_FAM_AGNC
[2023-06-29T22:01:11.300+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:01:13.612+0000] {logging_mixin.py:149} INFO - Attempting to get max value by column ID on table PR_LRS.FOSTER_FAM_AGNC
[2023-06-29T22:01:13.612+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:01:14.990+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table...
[2023-06-29T22:01:15.581+0000] {logging_mixin.py:149} INFO - Filter Processor Disabled
[2023-06-29T22:01:15.581+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:01:16.677+0000] {logging_mixin.py:149} INFO - 34262
[2023-06-29T22:01:16.677+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:01:17.917+0000] {logging_mixin.py:149} INFO - Checking if HDFS Directory /dev/FOSTER_FAM_AGNC exists...
[2023-06-29T22:01:17.917+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:01:17.926+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:01:17.926+0000] {client.py:320} INFO - Fetching status for '/dev/FOSTER_FAM_AGNC'.
[2023-06-29T22:01:17.931+0000] {logging_mixin.py:149} INFO - HDFS Path Exists
[2023-06-29T22:01:17.931+0000] {logging_mixin.py:149} INFO - Should go in here if table dir exists
[2023-06-29T22:01:19.790+0000] {logging_mixin.py:149} INFO - read dir table should not fail and go in here...
[2023-06-29T22:01:27.326+0000] {logging_mixin.py:149} INFO - no rows to upsert on table FOSTER_FAM_AGNC
[2023-06-29T22:01:30.217+0000] {logging_mixin.py:149} INFO - Updating rows on table FOSTER_FAM_AGNC
[2023-06-29T22:01:41.818+0000] {logging_mixin.py:149} INFO - no rows to delete on table FOSTER_FAM_AGNC
[2023-06-29T22:01:41.819+0000] {logging_mixin.py:149} INFO - Table Processing complete for PR_LRS.FOSTER_FAM_AGNC
[2023-06-29T22:01:41.819+0000] {logging_mixin.py:149} INFO - processing 2 out of 48
[2023-06-29T22:01:41.819+0000] {logging_mixin.py:149} INFO - Attempting to get lowest value by column ID on table PR_LRS.EBT_EXPG_EXCPT
[2023-06-29T22:01:41.819+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:01:42.861+0000] {logging_mixin.py:149} INFO - Attempting to get max value by column ID on table PR_LRS.EBT_EXPG_EXCPT
[2023-06-29T22:01:42.861+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:01:43.996+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table...
[2023-06-29T22:01:44.590+0000] {logging_mixin.py:149} INFO - Filter Processor Disabled
[2023-06-29T22:01:44.590+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:01:45.873+0000] {logging_mixin.py:149} INFO - 316506
[2023-06-29T22:01:45.873+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:01:47.152+0000] {logging_mixin.py:149} INFO - Checking if HDFS Directory /dev/EBT_EXPG_EXCPT exists...
[2023-06-29T22:01:47.152+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:01:47.160+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:01:47.160+0000] {client.py:320} INFO - Fetching status for '/dev/EBT_EXPG_EXCPT'.
[2023-06-29T22:01:47.164+0000] {logging_mixin.py:149} INFO - HDFS Path Exists
[2023-06-29T22:01:47.164+0000] {logging_mixin.py:149} INFO - Should go in here if table dir exists
[2023-06-29T22:01:47.273+0000] {logging_mixin.py:149} INFO - read dir table should not fail and go in here...
[2023-06-29T22:02:14.135+0000] {logging_mixin.py:149} INFO - no rows to upsert on table EBT_EXPG_EXCPT
[2023-06-29T22:02:40.434+0000] {logging_mixin.py:149} INFO - Updating rows on table EBT_EXPG_EXCPT
[2023-06-29T22:03:35.168+0000] {logging_mixin.py:149} INFO - no rows to delete on table EBT_EXPG_EXCPT
[2023-06-29T22:03:35.169+0000] {logging_mixin.py:149} INFO - Table Processing complete for PR_LRS.EBT_EXPG_EXCPT
[2023-06-29T22:03:35.169+0000] {logging_mixin.py:149} INFO - processing 3 out of 48
[2023-06-29T22:03:35.169+0000] {logging_mixin.py:149} INFO - Attempting to get lowest value by column ID on table PR_LRS.ENCL_ESTIMATE
[2023-06-29T22:03:35.169+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:03:36.259+0000] {logging_mixin.py:149} INFO - Attempting to get max value by column ID on table PR_LRS.ENCL_ESTIMATE
[2023-06-29T22:03:36.259+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:03:37.616+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table...
[2023-06-29T22:03:38.189+0000] {logging_mixin.py:149} INFO - Filter Processor Disabled
[2023-06-29T22:03:38.190+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:03:39.245+0000] {logging_mixin.py:149} INFO - 25979
[2023-06-29T22:03:39.245+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:03:40.335+0000] {logging_mixin.py:149} INFO - Checking if HDFS Directory /dev/ENCL_ESTIMATE exists...
[2023-06-29T22:03:40.335+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:03:40.349+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:03:40.349+0000] {client.py:320} INFO - Fetching status for '/dev/ENCL_ESTIMATE'.
[2023-06-29T22:03:40.354+0000] {logging_mixin.py:149} INFO - HDFS Path Exists
[2023-06-29T22:03:40.354+0000] {logging_mixin.py:149} INFO - Should go in here if table dir exists
[2023-06-29T22:03:40.463+0000] {logging_mixin.py:149} INFO - read dir table should not fail and go in here...
[2023-06-29T22:03:42.755+0000] {logging_mixin.py:149} INFO - no rows to upsert on table ENCL_ESTIMATE
[2023-06-29T22:03:44.636+0000] {logging_mixin.py:149} INFO - Updating rows on table ENCL_ESTIMATE
[2023-06-29T22:03:50.926+0000] {logging_mixin.py:149} INFO - no rows to delete on table ENCL_ESTIMATE
[2023-06-29T22:03:50.926+0000] {logging_mixin.py:149} INFO - Table Processing complete for PR_LRS.ENCL_ESTIMATE
[2023-06-29T22:03:50.927+0000] {logging_mixin.py:149} INFO - processing 4 out of 48
[2023-06-29T22:03:50.927+0000] {logging_mixin.py:149} INFO - Attempting to get lowest value by column ID on table PR_LRS.UNIT
[2023-06-29T22:03:50.927+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:03:52.027+0000] {logging_mixin.py:149} INFO - Attempting to get max value by column ID on table PR_LRS.UNIT
[2023-06-29T22:03:52.027+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:03:53.151+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table...
[2023-06-29T22:03:53.696+0000] {logging_mixin.py:149} INFO - Filter Processor Disabled
[2023-06-29T22:03:53.696+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:03:54.800+0000] {logging_mixin.py:149} INFO - 42879
[2023-06-29T22:03:54.801+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:03:55.911+0000] {logging_mixin.py:149} INFO - Checking if HDFS Directory /dev/UNIT exists...
[2023-06-29T22:03:55.911+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:03:55.919+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:03:55.919+0000] {client.py:320} INFO - Fetching status for '/dev/UNIT'.
[2023-06-29T22:03:55.923+0000] {logging_mixin.py:149} INFO - HDFS Path Exists
[2023-06-29T22:03:55.923+0000] {logging_mixin.py:149} INFO - Should go in here if table dir exists
[2023-06-29T22:03:56.025+0000] {logging_mixin.py:149} INFO - read dir table should not fail and go in here...
[2023-06-29T22:04:04.266+0000] {logging_mixin.py:149} INFO - no rows to upsert on table UNIT
[2023-06-29T22:04:08.246+0000] {logging_mixin.py:149} INFO - Updating rows on table UNIT
[2023-06-29T22:04:16.921+0000] {logging_mixin.py:149} INFO - no rows to delete on table UNIT
[2023-06-29T22:04:16.921+0000] {logging_mixin.py:149} INFO - Table Processing complete for PR_LRS.UNIT
[2023-06-29T22:04:16.921+0000] {logging_mixin.py:149} INFO - processing 5 out of 48
[2023-06-29T22:04:16.921+0000] {logging_mixin.py:149} INFO - Attempting to get lowest value by column ID on table PR_LRS.IEVS_SAVE_EXCEPT
[2023-06-29T22:04:16.921+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:04:18.102+0000] {logging_mixin.py:149} INFO - Attempting to get max value by column ID on table PR_LRS.IEVS_SAVE_EXCEPT
[2023-06-29T22:04:18.102+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:04:19.206+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table...
[2023-06-29T22:04:19.733+0000] {logging_mixin.py:149} INFO - Filter Processor Disabled
[2023-06-29T22:04:19.733+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:04:20.993+0000] {logging_mixin.py:149} INFO - 397776
[2023-06-29T22:04:20.993+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:04:22.325+0000] {logging_mixin.py:149} INFO - Checking if HDFS Directory /dev/IEVS_SAVE_EXCEPT exists...
[2023-06-29T22:04:22.325+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:04:22.332+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:04:22.332+0000] {client.py:320} INFO - Fetching status for '/dev/IEVS_SAVE_EXCEPT'.
[2023-06-29T22:04:22.335+0000] {logging_mixin.py:149} INFO - HDFS Path Exists
[2023-06-29T22:04:22.335+0000] {logging_mixin.py:149} INFO - Should go in here if table dir exists
[2023-06-29T22:04:22.411+0000] {logging_mixin.py:149} INFO - read dir table should not fail and go in here...
[2023-06-29T22:04:57.917+0000] {logging_mixin.py:149} INFO - no rows to upsert on table IEVS_SAVE_EXCEPT
[2023-06-29T22:05:37.154+0000] {logging_mixin.py:149} INFO - Updating rows on table IEVS_SAVE_EXCEPT
[2023-06-29T22:07:07.756+0000] {logging_mixin.py:149} INFO - no rows to delete on table IEVS_SAVE_EXCEPT
[2023-06-29T22:07:07.756+0000] {logging_mixin.py:149} INFO - Table Processing complete for PR_LRS.IEVS_SAVE_EXCEPT
[2023-06-29T22:07:07.757+0000] {logging_mixin.py:149} INFO - processing 6 out of 48
[2023-06-29T22:07:07.757+0000] {logging_mixin.py:149} INFO - Attempting to get lowest value by column ID on table PR_LRS.WTW_24_MONTH_SIGN_DATE
[2023-06-29T22:07:07.757+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:07:08.912+0000] {logging_mixin.py:149} INFO - Attempting to get max value by column ID on table PR_LRS.WTW_24_MONTH_SIGN_DATE
[2023-06-29T22:07:08.912+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:07:09.956+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table...
[2023-06-29T22:07:10.446+0000] {logging_mixin.py:149} INFO - Filter Processor Disabled
[2023-06-29T22:07:10.447+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:07:11.864+0000] {logging_mixin.py:149} INFO - 830945
[2023-06-29T22:07:11.864+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:07:13.158+0000] {logging_mixin.py:149} INFO - Checking if HDFS Directory /dev/WTW_24_MONTH_SIGN_DATE exists...
[2023-06-29T22:07:13.158+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:07:13.167+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:07:13.167+0000] {client.py:320} INFO - Fetching status for '/dev/WTW_24_MONTH_SIGN_DATE'.
[2023-06-29T22:07:13.171+0000] {logging_mixin.py:149} INFO - HDFS Path Exists
[2023-06-29T22:07:13.171+0000] {logging_mixin.py:149} INFO - Should go in here if table dir exists
[2023-06-29T22:07:13.265+0000] {logging_mixin.py:149} INFO - read dir table should not fail and go in here...
[2023-06-29T22:07:56.488+0000] {logging_mixin.py:149} INFO - no rows to upsert on table WTW_24_MONTH_SIGN_DATE
[2023-06-29T22:08:48.344+0000] {logging_mixin.py:149} INFO - Updating rows on table WTW_24_MONTH_SIGN_DATE
[2023-06-29T22:10:37.388+0000] {logging_mixin.py:149} INFO - no rows to delete on table WTW_24_MONTH_SIGN_DATE
[2023-06-29T22:10:37.388+0000] {logging_mixin.py:149} INFO - Table Processing complete for PR_LRS.WTW_24_MONTH_SIGN_DATE
[2023-06-29T22:10:37.389+0000] {logging_mixin.py:149} INFO - processing 7 out of 48
[2023-06-29T22:10:37.389+0000] {logging_mixin.py:149} INFO - Attempting to get lowest value by column ID on table PR_LRS.RULES_ADMIN
[2023-06-29T22:10:37.389+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:10:38.564+0000] {logging_mixin.py:149} INFO - Attempting to get max value by column ID on table PR_LRS.RULES_ADMIN
[2023-06-29T22:10:38.564+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:10:39.689+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table...
[2023-06-29T22:10:40.229+0000] {logging_mixin.py:149} INFO - Filter Processor Disabled
[2023-06-29T22:10:40.230+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:10:41.433+0000] {logging_mixin.py:149} INFO - 25885
[2023-06-29T22:10:41.433+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:10:42.564+0000] {logging_mixin.py:149} INFO - Checking if HDFS Directory /dev/RULES_ADMIN exists...
[2023-06-29T22:10:42.564+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:10:42.575+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:10:42.575+0000] {client.py:320} INFO - Fetching status for '/dev/RULES_ADMIN'.
[2023-06-29T22:10:42.580+0000] {logging_mixin.py:149} INFO - HDFS Path Exists
[2023-06-29T22:10:42.580+0000] {logging_mixin.py:149} INFO - Should go in here if table dir exists
[2023-06-29T22:10:42.668+0000] {logging_mixin.py:149} INFO - read dir table should not fail and go in here...
[2023-06-29T22:10:44.925+0000] {logging_mixin.py:149} INFO - no rows to upsert on table RULES_ADMIN
[2023-06-29T22:10:46.446+0000] {logging_mixin.py:149} INFO - Updating rows on table RULES_ADMIN
[2023-06-29T22:10:51.896+0000] {logging_mixin.py:149} INFO - no rows to delete on table RULES_ADMIN
[2023-06-29T22:10:51.897+0000] {logging_mixin.py:149} INFO - Table Processing complete for PR_LRS.RULES_ADMIN
[2023-06-29T22:10:51.897+0000] {logging_mixin.py:149} INFO - processing 8 out of 48
[2023-06-29T22:10:51.897+0000] {logging_mixin.py:149} INFO - Attempting to get lowest value by column HST_ID on table PR_LRS.COUNTY_PARAMTR_ADMIN_HST
[2023-06-29T22:10:51.897+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:10:52.997+0000] {logging_mixin.py:149} INFO - Attempting to get max value by column HST_ID on table PR_LRS.COUNTY_PARAMTR_ADMIN_HST
[2023-06-29T22:10:52.997+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:10:54.067+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table...
[2023-06-29T22:10:54.637+0000] {logging_mixin.py:149} INFO - Filter Processor Disabled
[2023-06-29T22:10:54.637+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:10:55.721+0000] {logging_mixin.py:149} INFO - 48818
[2023-06-29T22:10:55.721+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:10:56.851+0000] {logging_mixin.py:149} INFO - Checking if HDFS Directory /dev/COUNTY_PARAMTR_ADMIN_HST exists...
[2023-06-29T22:10:56.851+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:10:56.859+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:10:56.859+0000] {client.py:320} INFO - Fetching status for '/dev/COUNTY_PARAMTR_ADMIN_HST'.
[2023-06-29T22:10:56.863+0000] {logging_mixin.py:149} INFO - HDFS Path Exists
[2023-06-29T22:10:56.863+0000] {logging_mixin.py:149} INFO - Should go in here if table dir exists
[2023-06-29T22:10:56.951+0000] {logging_mixin.py:149} INFO - read dir table should not fail and go in here...
[2023-06-29T22:11:01.366+0000] {logging_mixin.py:149} INFO - no rows to upsert on table COUNTY_PARAMTR_ADMIN_HST
[2023-06-29T22:11:04.689+0000] {logging_mixin.py:149} INFO - Updating rows on table COUNTY_PARAMTR_ADMIN_HST
[2023-06-29T22:11:13.278+0000] {logging_mixin.py:149} INFO - no rows to delete on table COUNTY_PARAMTR_ADMIN_HST
[2023-06-29T22:11:13.278+0000] {logging_mixin.py:149} INFO - Table Processing complete for PR_LRS.COUNTY_PARAMTR_ADMIN_HST
[2023-06-29T22:11:13.278+0000] {logging_mixin.py:149} INFO - processing 9 out of 48
[2023-06-29T22:11:13.278+0000] {logging_mixin.py:149} INFO - Attempting to get lowest value by column ID on table PR_LRS.C4Y_PREG
[2023-06-29T22:11:13.278+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:11:14.451+0000] {logging_mixin.py:149} INFO - Attempting to get max value by column ID on table PR_LRS.C4Y_PREG
[2023-06-29T22:11:14.451+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:11:15.602+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table...
[2023-06-29T22:11:16.137+0000] {logging_mixin.py:149} INFO - Filter Processor Disabled
[2023-06-29T22:11:16.138+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:11:17.190+0000] {logging_mixin.py:149} INFO - 87659
[2023-06-29T22:11:17.190+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:11:18.320+0000] {logging_mixin.py:149} INFO - Checking if HDFS Directory /dev/C4Y_PREG exists...
[2023-06-29T22:11:18.320+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:11:18.328+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:11:18.328+0000] {client.py:320} INFO - Fetching status for '/dev/C4Y_PREG'.
[2023-06-29T22:11:18.332+0000] {logging_mixin.py:149} INFO - HDFS Path Exists
[2023-06-29T22:11:18.333+0000] {logging_mixin.py:149} INFO - Should go in here if table dir exists
[2023-06-29T22:11:18.434+0000] {logging_mixin.py:149} INFO - read dir table should not fail and go in here...
[2023-06-29T22:11:23.099+0000] {logging_mixin.py:149} INFO - no rows to upsert on table C4Y_PREG
[2023-06-29T22:11:26.820+0000] {logging_mixin.py:149} INFO - Updating rows on table C4Y_PREG
[2023-06-29T22:11:38.679+0000] {logging_mixin.py:149} INFO - no rows to delete on table C4Y_PREG
[2023-06-29T22:11:38.679+0000] {logging_mixin.py:149} INFO - Table Processing complete for PR_LRS.C4Y_PREG
[2023-06-29T22:11:38.680+0000] {logging_mixin.py:149} INFO - processing 10 out of 48
[2023-06-29T22:11:38.680+0000] {logging_mixin.py:149} INFO - Attempting to get lowest value by column ID on table PR_LRS.IEVS_PRISON_MATCH
[2023-06-29T22:11:38.680+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:11:39.724+0000] {logging_mixin.py:149} INFO - Attempting to get max value by column ID on table PR_LRS.IEVS_PRISON_MATCH
[2023-06-29T22:11:39.724+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:11:40.770+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table...
[2023-06-29T22:11:41.293+0000] {logging_mixin.py:149} INFO - Filter Processor Disabled
[2023-06-29T22:11:41.293+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:11:42.612+0000] {logging_mixin.py:149} INFO - 432367
[2023-06-29T22:11:42.612+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:11:43.917+0000] {logging_mixin.py:149} INFO - Checking if HDFS Directory /dev/IEVS_PRISON_MATCH exists...
[2023-06-29T22:11:43.917+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:11:43.924+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:11:43.924+0000] {client.py:320} INFO - Fetching status for '/dev/IEVS_PRISON_MATCH'.
[2023-06-29T22:11:43.929+0000] {logging_mixin.py:149} INFO - HDFS Path Exists
[2023-06-29T22:11:43.929+0000] {logging_mixin.py:149} INFO - Should go in here if table dir exists
[2023-06-29T22:11:44.036+0000] {logging_mixin.py:149} INFO - read dir table should not fail and go in here...
[2023-06-29T22:12:26.560+0000] {logging_mixin.py:149} INFO - no rows to upsert on table IEVS_PRISON_MATCH
[2023-06-29T22:13:09.296+0000] {logging_mixin.py:149} INFO - Updating rows on table IEVS_PRISON_MATCH
[2023-06-29T22:14:37.642+0000] {logging_mixin.py:149} INFO - no rows to delete on table IEVS_PRISON_MATCH
[2023-06-29T22:14:37.643+0000] {logging_mixin.py:149} INFO - Table Processing complete for PR_LRS.IEVS_PRISON_MATCH
[2023-06-29T22:14:37.643+0000] {logging_mixin.py:149} INFO - processing 11 out of 48
[2023-06-29T22:14:37.643+0000] {logging_mixin.py:149} INFO - Attempting to get lowest value by column ID on table PR_LRS.ORG_ACCT
[2023-06-29T22:14:37.643+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:14:38.751+0000] {logging_mixin.py:149} INFO - Attempting to get max value by column ID on table PR_LRS.ORG_ACCT
[2023-06-29T22:14:38.751+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:14:39.877+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table...
[2023-06-29T22:14:40.411+0000] {logging_mixin.py:149} INFO - Filter Processor Disabled
[2023-06-29T22:14:40.412+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:14:41.500+0000] {logging_mixin.py:149} INFO - 30003
[2023-06-29T22:14:41.500+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:14:42.673+0000] {logging_mixin.py:149} INFO - Checking if HDFS Directory /dev/ORG_ACCT exists...
[2023-06-29T22:14:42.673+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:14:42.680+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:14:42.681+0000] {client.py:320} INFO - Fetching status for '/dev/ORG_ACCT'.
[2023-06-29T22:14:42.684+0000] {logging_mixin.py:149} INFO - HDFS Path Exists
[2023-06-29T22:14:42.684+0000] {logging_mixin.py:149} INFO - Should go in here if table dir exists
[2023-06-29T22:14:42.769+0000] {logging_mixin.py:149} INFO - read dir table should not fail and go in here...
[2023-06-29T22:14:46.156+0000] {logging_mixin.py:149} INFO - no rows to upsert on table ORG_ACCT
[2023-06-29T22:14:49.414+0000] {logging_mixin.py:149} INFO - Updating rows on table ORG_ACCT
[2023-06-29T22:14:55.600+0000] {logging_mixin.py:149} INFO - no rows to delete on table ORG_ACCT
[2023-06-29T22:14:55.600+0000] {logging_mixin.py:149} INFO - Table Processing complete for PR_LRS.ORG_ACCT
[2023-06-29T22:14:55.600+0000] {logging_mixin.py:149} INFO - processing 12 out of 48
[2023-06-29T22:14:55.600+0000] {logging_mixin.py:149} INFO - Attempting to get lowest value by column ID on table PR_LRS.TRAIN_PGM
[2023-06-29T22:14:55.600+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:14:56.706+0000] {logging_mixin.py:149} INFO - Attempting to get max value by column ID on table PR_LRS.TRAIN_PGM
[2023-06-29T22:14:56.706+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:14:57.821+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table...
[2023-06-29T22:14:58.352+0000] {logging_mixin.py:149} INFO - Filter Processor Disabled
[2023-06-29T22:14:58.352+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:15:00.225+0000] {logging_mixin.py:149} INFO - 18600
[2023-06-29T22:15:00.225+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:15:01.308+0000] {logging_mixin.py:149} INFO - Checking if HDFS Directory /dev/TRAIN_PGM exists...
[2023-06-29T22:15:01.308+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:15:01.315+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:15:01.315+0000] {client.py:320} INFO - Fetching status for '/dev/TRAIN_PGM'.
[2023-06-29T22:15:01.318+0000] {logging_mixin.py:149} INFO - HDFS Path Exists
[2023-06-29T22:15:01.318+0000] {logging_mixin.py:149} INFO - Should go in here if table dir exists
[2023-06-29T22:15:01.423+0000] {logging_mixin.py:149} INFO - read dir table should not fail and go in here...
[2023-06-29T22:15:03.634+0000] {logging_mixin.py:149} INFO - no rows to upsert on table TRAIN_PGM
[2023-06-29T22:15:05.002+0000] {logging_mixin.py:149} INFO - Updating rows on table TRAIN_PGM
[2023-06-29T22:15:09.745+0000] {logging_mixin.py:149} INFO - no rows to delete on table TRAIN_PGM
[2023-06-29T22:15:09.745+0000] {logging_mixin.py:149} INFO - Table Processing complete for PR_LRS.TRAIN_PGM
[2023-06-29T22:15:09.746+0000] {logging_mixin.py:149} INFO - processing 13 out of 48
[2023-06-29T22:15:09.746+0000] {logging_mixin.py:149} INFO - Attempting to get lowest value by column ID on table PR_LRS.IFD_SSI
[2023-06-29T22:15:09.746+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:15:10.866+0000] {logging_mixin.py:149} INFO - Attempting to get max value by column ID on table PR_LRS.IFD_SSI
[2023-06-29T22:15:10.866+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:15:11.960+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table...
[2023-06-29T22:15:12.488+0000] {logging_mixin.py:149} INFO - Filter Processor Disabled
[2023-06-29T22:15:12.489+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:15:13.900+0000] {logging_mixin.py:149} INFO - 775890
[2023-06-29T22:15:13.900+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:15:15.261+0000] {logging_mixin.py:149} INFO - Checking if HDFS Directory /dev/IFD_SSI exists...
[2023-06-29T22:15:15.261+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:15:15.267+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:15:15.267+0000] {client.py:320} INFO - Fetching status for '/dev/IFD_SSI'.
[2023-06-29T22:15:15.270+0000] {logging_mixin.py:149} INFO - HDFS Path Exists
[2023-06-29T22:15:15.270+0000] {logging_mixin.py:149} INFO - Should go in here if table dir exists
[2023-06-29T22:15:15.356+0000] {logging_mixin.py:149} INFO - read dir table should not fail and go in here...
[2023-06-29T22:16:25.003+0000] {logging_mixin.py:149} INFO - no rows to upsert on table IFD_SSI
[2023-06-29T22:17:22.313+0000] {logging_mixin.py:149} INFO - Updating rows on table IFD_SSI
[2023-06-29T22:18:32.455+0000] {local_task_job_runner.py:291} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2023-06-29T22:18:32.461+0000] {process_utils.py:131} INFO - Sending Signals.SIGTERM to group 3773. PIDs of all processes in the group: [3774, 3773]
[2023-06-29T22:18:32.462+0000] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 3773
[2023-06-29T22:18:32.462+0000] {taskinstance.py:1517} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-06-29T22:18:32.462+0000] {clientserver.py:538} INFO - Error while receiving.
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/py4j/clientserver.py", line 511, in send_command
    answer = smart_decode(self.stream.readline()[:-1])
  File "/usr/local/lib/python3.10/socket.py", line 705, in readinto
    return self._sock.recv_into(b)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 1519, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2023-06-29T22:18:32.490+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-06-29T22:18:32.490+0000] {java_gateway.py:1055} ERROR - Exception while sending command.
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/py4j/clientserver.py", line 511, in send_command
    answer = smart_decode(self.stream.readline()[:-1])
  File "/usr/local/lib/python3.10/socket.py", line 705, in readinto
    return self._sock.recv_into(b)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 1519, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/py4j/java_gateway.py", line 1038, in send_command
    response = connection.send_command(command)
  File "/home/airflow/.local/lib/python3.10/site-packages/py4j/clientserver.py", line 539, in send_command
    raise Py4JNetworkError(
py4j.protocol.Py4JNetworkError: Error while sending or receiving
[2023-06-29T22:18:32.492+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-06-29T22:18:32.492+0000] {logging_mixin.py:149} INFO - Error Processing Table PR_LRS.IFD_SSI:An error occurred while calling o884.execute
[2023-06-29T22:18:32.492+0000] {logging_mixin.py:149} INFO - processing 13 out of 48
[2023-06-29T22:18:32.492+0000] {logging_mixin.py:149} INFO - Attempting to get lowest value by column ID on table PR_LRS.C4Y_PERS_NON_COMPL
[2023-06-29T22:18:32.492+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:18:33.272+0000] {clientserver.py:538} INFO - Error while receiving.
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/py4j/clientserver.py", line 516, in send_command
    raise Py4JNetworkError("Answer from Java side is empty")
py4j.protocol.Py4JNetworkError: Answer from Java side is empty
[2023-06-29T22:18:33.273+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-06-29T22:18:33.274+0000] {java_gateway.py:1055} ERROR - Exception while sending command.
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/py4j/clientserver.py", line 516, in send_command
    raise Py4JNetworkError("Answer from Java side is empty")
py4j.protocol.Py4JNetworkError: Answer from Java side is empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/py4j/java_gateway.py", line 1038, in send_command
    response = connection.send_command(command)
  File "/home/airflow/.local/lib/python3.10/site-packages/py4j/clientserver.py", line 539, in send_command
    raise Py4JNetworkError(
py4j.protocol.Py4JNetworkError: Error while sending or receiving
[2023-06-29T22:18:33.274+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-06-29T22:18:33.274+0000] {logging_mixin.py:149} INFO - Error in reading oracle table by query: An error occurred while calling o890.load
[2023-06-29T22:18:33.274+0000] {logging_mixin.py:149} INFO - Could not get min count: 'NoneType' object has no attribute 'head'
[2023-06-29T22:18:33.274+0000] {logging_mixin.py:149} INFO - Attempting to get max value by column ID on table PR_LRS.C4Y_PERS_NON_COMPL
[2023-06-29T22:18:33.274+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table by query...
[2023-06-29T22:18:33.274+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-06-29T22:18:33.274+0000] {logging_mixin.py:149} INFO - Error in reading oracle table by query: [Errno 111] Connection refused
[2023-06-29T22:18:33.274+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-06-29T22:18:33.274+0000] {logging_mixin.py:149} INFO - Could not get max count: 'NoneType' object has no attribute 'head'
[2023-06-29T22:18:33.274+0000] {logging_mixin.py:149} INFO - could not reliablily get upper and lower bounds of table PR_LRS.C4Y_PERS_NON_COMPL, reading without partition 
[2023-06-29T22:18:33.274+0000] {logging_mixin.py:149} INFO - Attempting to read oracle table...
[2023-06-29T22:18:33.275+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-06-29T22:18:33.275+0000] {logging_mixin.py:149} INFO - Read on oracle table failed: [Errno 111] Connection refused
[2023-06-29T22:18:33.275+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-06-29T22:18:33.275+0000] {logging_mixin.py:149} INFO - Filter Processor Disabled
[2023-06-29T22:18:33.275+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:18:33.275+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-06-29T22:18:33.275+0000] {logging_mixin.py:149} INFO - Error in reading oracle table by rowcount: [Errno 111] Connection refused
[2023-06-29T22:18:33.275+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-06-29T22:18:33.275+0000] {logging_mixin.py:149} INFO - None
[2023-06-29T22:18:33.275+0000] {logging_mixin.py:149} INFO - Attempting to read oracle rowcount by query...
[2023-06-29T22:18:33.275+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-06-29T22:18:33.275+0000] {logging_mixin.py:149} INFO - Error in reading oracle table by rowcount: [Errno 111] Connection refused
[2023-06-29T22:18:33.275+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-06-29T22:18:33.275+0000] {logging_mixin.py:149} INFO - Checking if HDFS Directory /dev/C4Y_PERS_NON_COMPL exists...
[2023-06-29T22:18:33.275+0000] {client.py:1123} INFO - Listing '/'.
[2023-06-29T22:18:33.283+0000] {logging_mixin.py:149} INFO - HDFS Connection Successfull
[2023-06-29T22:18:33.283+0000] {client.py:320} INFO - Fetching status for '/dev/C4Y_PERS_NON_COMPL'.
[2023-06-29T22:18:33.287+0000] {logging_mixin.py:149} INFO - HDFS Path Exists
[2023-06-29T22:18:33.287+0000] {logging_mixin.py:149} INFO - Should go in here if table dir exists
[2023-06-29T22:18:33.287+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-06-29T22:18:33.287+0000] {logging_mixin.py:149} INFO - Error in reading dataframe:[Errno 111] Connection refused
[2023-06-29T22:18:33.287+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-06-29T22:18:33.287+0000] {logging_mixin.py:149} INFO - Error in writing dataframe:'NoneType' object has no attribute 'write'
[2023-06-29T22:18:33.288+0000] {logging_mixin.py:149} INFO - Table write error, skipping table
[2023-06-29T22:18:33.289+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-06-29T22:18:33.304+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=Oracle_to_Filestore, task_id=execute_function_task, execution_date=20230629T220102, start_date=20230629T220103, end_date=20230629T221833
[2023-06-29T22:18:33.344+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=3774, status='terminated', started='22:01:03') (3774) terminated with exit code None
[2023-06-29T22:18:33.345+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=3773, status='terminated', exitcode=0, started='22:01:03') (3773) terminated with exit code 0
